File: frontends/Julia/Exasim/src/exasim_path.jl
function exasim_path()
    cdir = pwd()
    idx = findlast("Exasim", cdir)
    idx === nothing && error("exasim_path: current directory $cdir does not contain 'Exasim' in its path.")
    return cdir[firstindex(cdir):last(idx)]
end

File: frontends/Python/exasim/exasim_path.py
import os


def exasim_path():
    cdir = os.getcwd()
    idx = cdir.rfind("Exasim")
    if idx < 0:
        raise RuntimeError(
            f"Current directory {cdir} does not contain 'Exasim' in its path.")
    return cdir[:idx + len("Exasim")]
